extendeddatacrossword.go

```diff
@@ -15,6 +15,8 @@ type Axis int
 const (
 	Row Axis = iota
 	Col
+
+	NoShareInsertion = -1
```

Comment:
Agreed that this should not to be exported. Everything else LGTM.

Reply:
Resolved